omero/sysadmins/unix/install-web/walkthrough/omeroweb-install-centos7-ice3.6.rst

```diff
@@ -82,7 +73,6 @@ Configuring OMERO.web
 
 For convenience the main OMERO.web configuration options have been defined as environment variables. You can either use your own values, or alternatively use the following ones::
 
-    export WEBSESSION=True
     export OMERODIR=/opt/omero/web/omero-web
     export WEBPORT=80
```

Comment:
You don't need to `export` `WEBPORT` or `WEBSERVER_NAME` because they're used in the same shell. `export` is only needed if they're used by a subprocess.

Reply:
extracting from the bash script automatically.